jax/numpy/lax_numpy.py

```diff
@@ -3088,6 +3088,19 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
   X_T = X.T if w is None else (X * w).T
   return true_divide(dot(X, X_T.conj()), f).squeeze()
 
+@_wraps(onp.convolve)
+def convolve(x, y, mode='full'):
+  if shape(y)[0] > shape(x)[0]:
```

Comment:
It might be a good idea to check that the inputs are 1D, and raise an error otherwise.